packages/stack/src/components/credential-sign-up.tsx

```diff
@@ -12,7 +12,7 @@ import { useStackApp } from "..";
 import { useTranslation } from "../lib/translations";
 import { FormWarningText } from "./elements/form-warning";
 
-export function CredentialSignUp() {
+export function CredentialSignUp(props: { noPasswordRepeat?: boolean }) {
   const { t } = useTranslation();
 
   const schema = yupObject({
```

Comment:
Did you test if this signs in? I think the schema checks is not modified so it should not pass the "password must match" validation